### PR TITLE
Monitoring grafana - Orion-LD dashboard

### PR DESCRIPTION
**Pull request by manuvaldi**

### CODE REVIEW

**Comment by wistefan** [comment]
The PR overall looks good to me, but why dont we simply use the grafana in openshift-monitoring?

File: applications/grafana-metrics/chart/templates/secret-datasources.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - name: Prometheus-Thanos
+        access: proxy
+        editable: true
+        isDefault: true
+        jsonData:
+          httpHeaderName1: 'Authorization'
+          timeInterval: 5s
+          tlsSkipVerify: true
+        secureJsonData:
+          httpHeaderValue1: 'Bearer $__file{/run/secrets/kubernetes.io/serviceaccount/token}'
```
Comment (jtudelag) [inline applications/grafana-metrics/chart/templates/secret-datasources.yaml:22]: Out of curiosity, how does this "$__file{}" work???

**Comment by manuvaldi** [comment]
> The PR overall looks good to me, but why dont we simply use the grafana in openshift-monitoring?

Because of custom dashboards

**Review by jtudelag — COMMENTED** [review]
LGTM

**Comment by jtudelag** [comment]
> The PR overall looks good to me, but why dont we simply use the grafana in openshift-monitoring?

Hey @wistefan good question! That Grafana is read-only, managed by the openshift-monitoring operators,so it is not possible to do that.

File: applications/grafana-metrics/chart/templates/secret-datasources.yaml (hunk 1 above)
Comment (manuvaldi) [reply applications/grafana-metrics/chart/templates/secret-datasources.yaml:22]: It's a provider to read passwords or secrets from files. Magic!! https://grafana.com/docs/grafana/latest/administration/configuration/#file-provider

**Comment by wistefan** [comment]
> > The PR overall looks good to me, but why dont we simply use the grafana in openshift-monitoring?
> 
> Hey @wistefan good question! That Grafana is read-only, managed by the openshift-monitoring operators,so it is not possible to do that.

There is an ugly, hacky way and I was hoping you guys show me the good path;) But I'm fine with an additional one.

**Pull request merged**